02-ca-certificates.md

```diff
@@ -21,14 +21,15 @@ Following the steps below you will result the certificate needed for Azure Appli
 
    Create a CA certificate for each Azure Application Gateway. You can use your company domain or try get a certificate for each domain using [Azure Subdomain Certificates Generation](./certificate-generation/README.md).
 
-   :warning: We are waiting for two certificates in the following example (Please, pay attention the regions). We called bu0001a004203 (eastus2) and bu0001a004204 (westus2) each subdomain, but the DNS values could be not available. In that case, you can change the following values.
+   :warning: We are waiting for two certificates in the following example (Please, pay attention the regions where the certs are needed). We called CLUSTER_SUBDOMAIN_03 and CLUSTER_SUBDOMAIN_04 each subdomain. You can change the name if you desire.
 
    ```bash
-   export CLUSTER_SUBDOMAIN_03=bu0001a004203
-   export CLUSTER_SUBDOMAIN_04=bu0001a004204
+   USER_NAME=$(az ad signed-in-user show --query 'mailNickname' -o tsv)
+   export CLUSTER_SUBDOMAIN_03=${USER_NAME//.}-region1
```

Comment:
I would let @ckittel decide what is better here. I think user names are ok. But any desire to use a a more random and repeatable (as far as I can see) solution using a number instead?

`export SUBDOMAIN_REGION1=$(bash -c 'RANDOM=$(az ad user show --id <e-Mail> --query immutableId -o tsv); echo region1-$RANDOM$RANDOM$RANDOM')`

it generate something like:

```output
region1-2221110329407
```

Reply:
based on @ckittel comments above, I think it might safe to proceed with this solution.

Reply:
yeah, just as long as it's natural and tied to the lifecycle of a _resource_ ... not a human.  Human relationships end with organizations (sometimes poorly) -- having some sort of value based on `employeeWeWishToForget` would be hilarious, but also terrible if someone actually does this literally ... :)  -- let's keep the humans out, and the robots/resources in :)